.travis/install.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if [ "$TYPE" = 'style' ]; then
+  sudo apt install shellcheck
```

Comment:
Thanks.  We use `apt-get` everywhere else, let's use that for consistency.   If there turn out to be advantages using `apt` over `apt-get`, we'll switch the whole project to use it (wiki, travis, etc).